lib/commands/project.js

```diff
@@ -65,6 +65,76 @@ function sortImages(images) {
   return images;
 }
 
+project.resize = function (projectName, servoSize, cb) {
+  var projectId, projectName;
+
+  projectController.find({
+      userId : userConfig.data.userId
+    },
+    function (err, projects) {
+      if (err) {
+        err = error.handleApiError(err, 'GET_PROJECTS', cb);
+        if (err.length > 0) {
+          return cb(err);
+        }
+      }
+      if (projects.length === 0) {
+        modulus.io.error('You currently have no projects. One can be created using the create command.');
+        return cb();
+      }
+      project.cmd = "resize";
```

Comment:
`project.cmd = "resize"` sets the cmd to be resize on a global object. After scanning for project.cmd, I found out that there is an "autoCmds" global array that does not contain/need "resize."

I removed this line.